deployment/helm/skaha/skaha-config/launch-notebook.yaml

```diff
@@ -86,6 +102,12 @@ spec:
           protocol: TCP
           name: notebook-port
         volumeMounts:
+        - mountPath: "/etc/passwd"
```

Comment:
In my testing I could not get a subpath mount of an emptydir volume to work, but I found documentation references suggesting it should work, so I am curious what specific issue might have been preventing this in my tests or if you were able to test and confirm that it works, if it is only supported in some k8s versions, etc.

Reply:
Also the volume mounts in the run container must be read-only.